src/6_branch.js

```diff
@@ -788,7 +793,8 @@ Branch.prototype['setIdentity'] = wrap(callback_params.CALLBACK_ERR_DATA, functi
 				safejson.parse(data['referring_data']) :
 				null;
 
-			session.patch(self._storage, { "identity": identity }, true);
+			// /v1/profile will return a new identity_id, but the same session_id
```

Comment:
We were already updating identity in local storage. This change updates identity_id as well in local storage. That ensures that we get the right identity_id at initialization, to send in /v1/open.

Reply:
ahh I see